src/main/java/de/unistuttgart/iste/gits/reward/service/calculation/FitnessScoreCalculator.java

```diff
@@ -22,198 +23,164 @@ public class FitnessScoreCalculator implements ScoreCalculator {
 
         for (Content content : contents) {
-            if (isDueForRepetition(content)) {
+            if (isDueForRepetition(content) && isNotNew(content)) {
                 int daysOverdue = calculateDaysOverdue(content);
-                double correctness = calculateCorrectnessModifier(content, today);
+                double correctness = calculateCorrectnessModifier(getLatestReview(content));
                 double decreasePerDay = 1 + (2 * daysOverdue * (1 - correctness));
                 fitnessDecrease += decreasePerDay;
-
-                System.out.println("Content: " + content.getId());
-                System.out.println("Days Overdue: " + daysOverdue);
-                System.out.println("Correctness: " + correctness);
-                System.out.println("Decrease Per Day: " + decreasePerDay);
             }
         }
 
-        System.out.println("Total Fitness Decrease: " + fitnessDecrease);
-
         return Math.min(MAX_DECREASE_PER_DAY, fitnessDecrease);
     }
 
     private boolean isDueForRepetition(Content content) {
         OffsetDateTime today = OffsetDateTime.now();
-        OffsetDateTime repetitionDate = content.getUserProgressData().getLastLearnDate();
-
-        System.out.println("Content: " + content.getId());
-        System.out.println("Today: " + today);
-        System.out.println("Repetition Date: " + repetitionDate);
+        OffsetDateTime repetitionDate = content.getUserProgressData().getNextLearnDate();
 
         // Check if the repetition date is today or in the past
-        boolean isDue = repetitionDate != null && (repetitionDate.isBefore(today) || repetitionDate.isEqual(today));
-
-        System.out.println("Is Due for Repetition: " + isDue);
+        return repetitionDate != null && (repetitionDate.isBefore(today) || repetitionDate.isEqual(today));
+    }
 
-        return isDue;
+    private boolean isNotNew(Content content) {
+        // check if the content has been learned before successfully
+        // otherwise it is considered new and should not be considered for fitness decrease
+        return content.getUserProgressData()
+                .getLog()
+                .stream()
+                .anyMatch(ProgressLogItem::getSuccess);
     }
 
+    private List<Content> getContentsToRepeat(List<Content> contents) {
+        return contents.stream()
+                .filter(this::isDueForRepetition)
+                .toList();
+    }
 
     private int calculateDaysOverdue(Content content) {
         OffsetDateTime today = OffsetDateTime.now();
         OffsetDateTime repetitionDate = content.getUserProgressData().getNextLearnDate();
 
-        System.out.println("Content: " + content.getId());
-        System.out.println("Today: " + today);
-        System.out.println("Repetition Date: " + repetitionDate);
-
-        // Calculate the number of days between the current date and the repetition date
-        long daysBetween = ChronoUnit.DAYS.between(today, repetitionDate);
```

Comment:
Don't use ChronoUnit.DAYS.between because here it returns a negative number, which messed up the whole calculation.  Duration.between(today, repetitionDate).abs().toDays(); is the way to go (abs() makes it an absolute value)